frappe_io/www/docs/user/en/tutorial/web-views.md

```diff
@@ -1,68 +0,0 @@
-<!-- base_template: frappe_io/www/frappe/frappe_base.html --><!-- add-breadcrumbs -->
```

Comment:
If someone visits this old page, they should be redirected to the new one.

Reply:
@netchampfaris is there a way to do redirects?